src/providers/anthropic/chatComplete.ts

```diff
@@ -118,6 +123,9 @@ export const AnthropicChatCompleteConfig: ProviderConfig = {
               const transformedMessage: Record<string, any> = {
                 role: msg.role,
                 content: [],
+                ...(msg.cache_control && {
```

Comment:
cache_control is only allowed inside the content array objects. So this mapping will not work as expected. It should be mapped inside individual objects